short_answer/static/js/src/short_answer.js

```diff
@@ -67,12 +75,18 @@ function ShortAnswerXBlock(runtime, element, options) {
 
     submissions.forEach(function(submission) {
       submission.answer = submission.answer || '';
-      if (submission.answered_at !== 'None') {
+      if (submission.answered_at && submission.answered_at !== 'None') {
```

Comment:
Can this `submission.answered_at !== 'None'` be replaced with something like `submission.answered_at` if we json.dump(None) on the server side? Not sure if we want to have None hardcoded on the JS side.

Reply:
In my test, when `answered_at` was not set you'd get `''` from the server, hence the first condition. I have no idea when 'None' is going to be sent back so I kept it for backwards compatibility.